api/db/schema.prisma

```diff
@@ -38,10 +39,10 @@ model Address {
   id            String     @id @default(cuid())
   restaurant    Restaurant @relation(fields: [restaurantId], references: [id])
   restaurantId  String     @unique
-  addressNumber String
-  addressStreet String
-  city          String
-  country       String
-  postalCode    String
-  state         String
+  addressNumber String?
```

Comment:
Since we put the address in sheets, let's get rid of anything in the DB related to addresses.